test/functional/mining_basic.py

```diff
@@ -92,11 +92,6 @@ def assert_submitblock(block, result_str_1, result_str_2=None):
 
         # round-trip the encoded bip34 block height commitment
         assert_equal(CScriptNum.decode(coinbase_tx.vin[0].scriptSig), next_height)
```

Comment:
This seems to be a test of the `create_coinbase()` function in blocktools.py. It might be worth moving it to a unit test there.